governance.md

```diff
@@ -1,36 +1,50 @@
 
-### Who controls the wp-cli.org site?
+The project roadmap is an organic, evolving vision developed between the project's maintainers, committers, and contributors. Generally, we try to make decisions in [alignment with the project's philosophy](https://make.wordpress.org/cli/handbook/philosophy/) and [committers credo](https://make.wordpress.org/cli/handbook/committers-credo/).
 
-The `wp-cli.org` domain currently owned by [andreascreten](https://github.com/andreascreten).
+## Why are some people members of the WP-CLI organization?
 
-The entire site is hosted on Github Pages, using Cloudflare for SSL:
+On occasion, we [grant write access to contributors](https://github.com/orgs/wp-cli/teams/committers) who have demonstrated, over a period of time, that they are capable and invested in moving the project forward. These committers are expected to:
 
-| **URL**                          | **Repository**                | **Branch** | **Updates**
-|:---------------------------------|:------------------------------|:-----------|:------------
-| http://wp-cli.org/               | [wp-cli/wp-cli.github.com][1] | master     | manual
-| http://wp-cli.org/package-index/ | [wp-cli/package-index][2]     | gh-pages   | cron job hosted on separate server (owned by daniel)
-| http://wp-cli.org/builds/        | [wp-cli/builds][3]            | gh-pages   | automatically by Travis CI
+* Hold product and code quality in the highest regard.
+* Exhibit stellar judgement and communication.
+* Participate with the project on a consistent basis.
 
-[1]: http://github.com/wp-cli/wp-cli.github.com
-[2]: http://github.com/wp-cli/package-index
-[3]: http://github.com/wp-cli/builds
+Becoming a committer can seem glamorous, but it also comes with expectations of responsibility, commitment, and humility.
 
-If you [subscribe for email updates](https://make.wordpress.org/cli/subscribe/), your email address is registered with WordPress.com through Automattic's Jetpack plugin.
+[andreascreten](https://github.com/andreascreten) is the original author of WP-CLI, and [scribu](https://github.com/scribu) was a long time maintainer.
+
+## What's the connection between WP-CLI and WordPress?
+
+As of December 2017, WP-CLI is a [formal WordPress project](https://make.wordpress.org/core/2016/12/28/supporting-the-future-of-wp-cli/).
+
+Practically-speaking, this means we get to take advantage of the WordPress project's people and software infrastructure:
+
+* Much of the WP-CLI documentation is hosted on the wordpress.org domain.
+* We get to use the `#cli` channel in the [WordPress Slack organization](https://make.wordpress.org/chat/).
+* If we need help with something, we can ask.
+* A couple core committers ([dd32](https://github.com/dd32) and [pento](https://github.com/pento)) have ownership permissions to the WP-CLI GitHub organization, in the unlikely event everyone else gets hit by a bus.
```

Comment:
couple of (missing "of"). Maybe "have" -> "has" though in my variant of English "have" is good (but in American English it's "has"?). Or maybe just "two" instead of "a couple of" to by-pass the issue!